Update feature list in README

kangalio opened the pull request.

kangalio committed:
Update feature list in README

arqunis commented on README.md:
```diff
@@ -161,14 +161,17 @@ the HTTP functions.
 - **unstable_discord_api**: Enables features of the Discord API that do not have a stable interface. The features might not have official documentation or are subject to change.
 - **simd_json**: Enables SIMD accelerated JSON parsing and rendering for API calls, use with `RUSTFLAGS="-C target-cpu=native"`
 - **temp_cache**: Enables temporary caching in functions that retrieve data via the HTTP API.
+- **chrono**: Uses the chrono crate to represent timestamps
+- **time**: Uses the time crate to represent timestamps
```
Might be worth mentioning these are mutually exclusive.

```suggestion
- **chrono**\*: Uses the chrono crate to represent timestamps
- **time**\*: Uses the time crate to represent timestamps

....

* -- Only one of these features may be active at a time.
```

kangalio replied:
Wait I just noticed the `time` feature is actually a no-op by now
![Screenshot_20230409_004707](https://user-images.githubusercontent.com/21220820/230745674-327d5430-d081-4378-b0e3-ac4dbae06b56.png)

Will remove the `time` line entirely then

kangalio committed:
Mention mutual-exlusitivity of time and chrono

kangalio committed:
Remove time feature (it's a no-op I realized)

kangalio committed:
Remove CI job for time feature

Time feature was a no-op because it's the default, hence all jobs (except the chrono one) already run with time, and we don't need to test it specifically anyways

mkrasnitski commented on README.md:
```diff
@@ -155,20 +155,21 @@ enough level that optional parameters can be provided at will via a JsonMap.
 - **unstable_discord_api**: Enables features of the Discord API that do not have a stable interface. The features might not have official documentation or are subject to change.
 - **simd_json**: Enables SIMD accelerated JSON parsing and rendering for API calls, use with `RUSTFLAGS="-C target-cpu=native"`
 - **temp_cache**: Enables temporary caching in functions that retrieve data via the HTTP API.
+- **chrono**: Uses the `chrono` crate to represent timestamps (instead of `time`, the default)
```
```suggestion
- **chrono**: Uses the `chrono` crate to represent timestamps. If disabled, the `time` crate is used instead.
```
This reads slightly weird, since `chrono` is a default feature.

kangalio committed:
Improve Chrono feature wording

Co-authored-by: Michael Krasnitski <42564254+mkrasnitski@users.noreply.github.com>
